### DISCUSSION TITLE
Google Photos JSON extract

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#2127] [Q&A] [answered]
Since Photoprism can read google takeout exports (specifically can recognize the annoying separate json files), is it possible for it to save the JSON info to the original photos exif. The documentation says that the fields are saved, but for me, they are not being saved on the original photo. How could I do this?

**Comment by graciousgrey** [comment, marked as answer]
Currently metadata read from JSON files or other sources is saved in the PhotoPrism database as well as in human readable export files (https://docs.photoprism.app/user-guide/advanced/export/).

PhotoPrism does not yet write metadata back to exif. That's planned for the future (https://github.com/photoprism/photoprism/issues/402).

**Reply by ghost** [reply]
Great! Thanks for your response!
